Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -2023,14 +2030,21 @@ public async Task UpdateImagesAsync(BaseItem item, bool forceUpdate = false)
                     image.Height = 0;
                 }
 
+                image.BlurHash = string.Empty;
                 try
                 {
-                    image.BlurHash = _imageProcessor.GetImageBlurHash(image.Path, size);
+                    if (_imageProcessor.CanComputeBlurHash(image.Path))
+                    {
+                        image.BlurHash = _imageProcessor.GetImageBlurHash(image.Path, size);
+                    }
+                    else
+                    {
+                        _logger.LogWarning("Cannot compute blurhash for {ImagePath}", image.Path);
```

Comment:
I doubt we want this warning every time an `.SVG` file is scanned. This is a normal case, and I know I wouldn't want to have to _keep_ wading through (just a different) message about BlurHashes on my many SVGs.

Reply:
I agree, this log level should be changed to `Debug` so it can still be viewed